Controllers/LoginScreen.swift

```diff
@@ -47,13 +38,25 @@ class LoginScreen: UsersDatabase, UITextFieldDelegate {
       staticIndex.firstLaunch = true
     }
   }
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    super.hideKeyboardWhenTappedAround()
+    super.moveViewWhenKeyboardAppears()
+
+    tfUsername.delegate = self
+    tfPassword.delegate = self
+    textView.text = "welcome_text".localized
+    forgottenPassword.setTitle("forgotten_password".localized, for: .normal)
+    forgottenPassword.underline()
+  }
 
   // MARK: - Navigation
 
   func checkLoggedStatus(token: String?) {
     if token == nil || !(authentificationService?.oauth2.hasUnexpiredAccessToken())! {
       if !Connectivity.isConnectedToInternet() {
-        performSegue(withIdentifier: "SegueNoInternetOnFirstConnection", sender: self)
+        performSegue(withIdentifier: "ShowNoInternetVC", sender: self)
```

Comment:
Segue identifiers should be written in camelCase, with a first letter lowercased